bot/commands/cmd_stats.go

```diff
@@ -0,0 +1,149 @@
+package commands
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/go-resty/resty"
+)
+
+type serverCore struct {
```

Comment:
You could import this from here: https://github.com/Southclaws/samp-servers-api/blob/master/types/server_core.go#L5-L14 instead of re-declaring it.